onadata/libs/utils/export_builder.py

```diff
@@ -153,6 +158,23 @@ def dict_to_joined_export(data, index, indices, name, survey, row,
     return output
 
 
+def track_task_progress(additions, total=None):
+    """
+    Updates the current export task with number of submission processed
+    :param additions:
+    :param total:
+    :return:
+    """
+    try:
+        meta = {'progress': additions}
+        if total:
+            meta.update({'total': total})
+        current_task.update_state(state='PENDING',
```

Comment:
Do we have any other started other than `pending`? For example, `started`? http://docs.celeryproject.org/en/latest/reference/celery.states.html#all-states